### ISSUE TITLE
How to combine softmax  and regression

### ISSUE THREAD

**Issue by nopattern** [apache/incubator-mxnet#8651]
I want to detect object and regress their position by heatmap. The object and background can be trained by softmax, but there should be no regression for background . I try to combine 2 by:
            regress_loss = mx.symbol.broadcast_mul(label, regression)   
where the label shape is (128,)   and the regression shape is (128,4,48,48) . 128 is the batch size. The error is 

> src/operator/tensor/./elemwise_binary_broadcast_op.h:66: Check failed: l == 1 || r == 1 operands could not be broadcast together with shapes (128,) (128,4,48,48)

so how can I to  set background(label==0)'s regression to zero?

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by vdantu** [comment]
@nopattern : Did you find answer to your question? It might be helpful to start this discussion on discuss.mxnet.io . You will find larger audience.

**Comment by nopattern** [comment]
@szha ,   @vdantu thank you,  I have solved the problem. sorry for forggeting to close.
